pageserve/app.py

```diff
@@ -21,12 +21,20 @@
 
 app = Flask(__name__)
 
+ENDPOINTS = ['USERS_ENDPOINT', 'EVENTS_ENDPOINT']
+for ENDPOINT in ENDPOINTS:
+    if ENDPOINT in os.environ:
+        app.config[ENDPOINT] = os.environ.get(ENDPOINT)
+    else:
+        raise NameError("Endpoint {} not defined.".format(ENDPOINT))
```

Comment:
Is this the best way to handle the env variables not being defined? Should this be a function instead? There will be at least one other endpoint.

Reply:
I think it would look a little cleaner (and have fewer globals) if you refactored into a function like:
```python
app = Flask(__name__)

config_endpoints(endpoints=['USERS_ENDPOINT', 'EVENTS_ENDPOINT'])

def config_endpoints(endpoints):
    ...
```

Reply:
Done in 377b8cf